### DISCUSSION TITLE
Combobox Group

### DISCUSSION THREAD

**Discussion by bciceriTwilio** [twilio-labs/paste#1366] [General] [answered]
Hi! We're working on updating the filters for the Account Insights page in Console. For that, we need the ability for users to create selections like, "Show me everything greater than X" or "Show me everything BUT transactions in this country." We also need the ability for users to easily remove the filters that they add. After exploring a few separate concepts, I've landed on this design for a grouped dropdown pattern:

![combobox](https://user-images.githubusercontent.com/82666366/115047539-3a981980-9e9e-11eb-9b7e-7b1ee24e5d4b.gif)
 
Are there any plans for a combobox group like this on the roadmap? Thanks!

**Comment by SiTaggart** [comment, marked as answer]
Hi @bciceriTwilio 

Connected form fields are on the roadmap, but they are currently not scheduled in any upcoming release right now.

You can keep an eye on the roadmap by visiting https://paste.twilio.design/roadmap

Depending on your business unit, your team via your design partner, should also be going through a regular quarterly road map review. This is where component requests are bubbled up to us and helps us continuously re-evaluate our roadmap based on immediate team needs over the coming quarters.
